eq-author/src/apollo/createApolloErrorLink.test.js

```diff
@@ -1,27 +1,83 @@
+import uuid from "uuid";
+import jwt from "jsonwebtoken";
 import { errorHandler } from "./createApolloErrorLink";
+import { sendSentryError, setSentryTag, setSentryUser } from "./sentryUtils";
 
-const dispatch = jest.fn();
-
-const createMockStore = () => ({
-  dispatch,
-});
+jest.mock("./sentryUtils");
 
 describe("createErrorLink", () => {
+  let dispatch;
+
+  const createAccessToken = (token, signingKey = uuid.v4()) => {
+    return jwt.sign(token, signingKey);
+  };
+
+  const createMockStore = () => ({
+    dispatch,
+  });
+
+  beforeAll(() => {
+    /* eslint-disable-next-line camelcase */
+    const tokenData = { user_id: "Test User", email: "test.user@email.com" };
+    const token = createAccessToken(tokenData);
+    localStorage.setItem("accessToken", token);
+
+    dispatch = jest.fn();
+  });
+
+  let networkError = {
+    fail: "Uh oh",
+  };
+  let graphQLErrors = [
+    {
+      fail: "Uh oh",
+    },
+  ];
+  let graphQLError = {
+    fail: "Uh oh",
+  };
+
   it("should dispatch a logout event when the server responds with a 401 error", () => {
     errorHandler(createMockStore, {
-      networkError: {
-        bodyText: "User does not exist",
-      },
+      networkError: { bodyText: "User does not exist" },
     });
     expect(dispatch).toHaveBeenCalledWith(expect.any(Function));
   });
 
   it("should dispatch a apiDownError when the server responds with a error", () => {
     errorHandler(createMockStore, {
-      networkError: {
-        fail: "Uh oh",
-      },
+      networkError,
     });
     expect(dispatch).toHaveBeenCalledWith({ type: "API_DOWN_ERROR" });
   });
+
+  it("should send network errors to sentry", () => {
+    errorHandler(createMockStore, {
+      networkError,
+    });
+    expect(sendSentryError).toHaveBeenCalledWith(networkError);
+  });
+
+  it("should send graphql errors to sentry", () => {
+    errorHandler(createMockStore, {
+      graphQLErrors,
+    });
+    expect(sendSentryError).toHaveBeenCalledWith(graphQLError);
```

Comment:
That object was never given to the errorHandler can you do something like this instead.
```suggestion
    expect(sendSentryError).toHaveBeenCalledWith(graphQLErrors[0]);
```